ltj/settings.py

```diff
@@ -44,8 +44,8 @@ def get_git_revision_hash():
     LOG_LEVEL=(str, "INFO"),
     SENTRY_DSN=(str, ""),
     SENTRY_ENVIRONMENT=(str, "unconfigured"),
-    WFS_SERVER_URL=(str, "http://localhost/"),
-    WFS_NAMESPACE=(str, "ltj-dev"),
+    WFS_SERVER_URL=(str, " https://kartta.hel.fi/ws/geoserver/avoindata/wfs"),
```

Comment:
Is it just me or is there a space between the `"` and the `h`

Reply:
Fixed now